### PR TITLE
FR - gender inclusive writing, more actionable content

### PR DESCRIPTION
**Pull request by abelards**
1. used genderless or gender-inclusive writing
   1a. changed sentences to avoid gendered french names or adjectives
   1b. when I could not do that, I added both forms (participant·e·s or orateurs·trices)
2. changed some anglicisms
   2a. Code of Conduct (code de conduite -> Code d'Éthique et de Déontologie) just once
   2b. materials (matériaux -> supports)
3. made things more explicit and actionable
   3a. oral comments -> oral or written comments
   3b. sanctions -> up to exclusion and legal action (already mentioned later in the document)
   3c. organisers only wear colored t-shirts -> they can have badges, armbands, lanyards...)

Concerning 3a &3b, I felt that:
- someone could say "my comment was made on Facebook, not orally".
- "from warnings to exclusion" -> anything they deem appropriate, up to exclusion and legal action
- contacting security and escorting -> it was unclear if organisers can just ask security to do it, or commit to doing it themselves if  no other solution can be found.

If anything, the slight changes in meaning here are demanding more commitment from the organisers.

### CODE REVIEW

**Comment by remy** [comment]
Are you able to also add the new introduction text that appears in the english version making clear that the CoC should be found on the event site?

**Comment by abelards** [comment]
@remy sure, here you are :)

File: index-fr.html [hunk 1]
```diff
@@ -14,80 +14,91 @@
     <h1>Code de conduite pour la conférence</h1>
 
     <p>
-      Tous les participants, conférenciers, sponsors et volontaires pour notre
-      conférence doivent accepter le code de conduite suivant. Les organisateurs
-      s'attacheront à faire respecter ce code durant l'événement. Nous attendons
-      de la part de chaque participant une coopération pour assurer un
-      environnement sain pour tous.
+      Toutes les personnes participant à la conférence,
+      y compris participant·e·s, orateurs·trices, sponsors et volontaires
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      y compris sponsors et bénévoles
```

Participant‧e‧s et orateurs‧rices sont attendus dans « participant à la conférence », ne pas les mentionner explicitement permet d'éviter l'écriture inclusive.

File: index-fr.html [hunk 2]
```diff
@@ -14,80 +14,91 @@
...
+      s'engagent à accepter le code d'éthique et de déontologie, ou "code de conduite" suivant.
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      s'engagent à accepter le code d'éthique et de déontologie, ou « code de conduite » suivant.
```

Guillemets français :wink:

File: index-fr.html [hunk 3]
```diff
@@ -14,80 +14,91 @@
...
+      L'organisation s'attachera à faire respecter ce code durant l'événement.
+      Nous attendons de la part de chacun·e leur coopération pour assurer un environnement sain pour tous.
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      Nous attendons la coopération de chacun‧e pour assurer un environnement sain pour tous.
```

File: index-fr.html [hunk 4]
```diff
@@ -14,80 +14,91 @@
...
     </p>
 
+    <div class="notice">
+      <h2>Important</h2>
+      <p>
+        Cette page est un modèle de code d'éthique et de déontologie (code de conduite) et
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
        Cette page est un modèle de code d'éthique et de déontologie (« code de conduite ») et
```

File: index-fr.html [hunk 5]
```diff
@@ -14,80 +14,91 @@
...
 
     <p>
       Les sponsors sont aussi sujet à la politique anti-harcèlement. En
-      particulier, les sponsors ne doivent pas utiliser d'images ou de matériels
-      à connotation sexuelle. Ils ne doivent pas non plus engager d'activités à
-      connotation sexuelle. L'équipe du stand (y compris les volontaires) ne
-      doivent pas utiliser de vêtements, uniformes ou costumes à connotation
+      particulier, les sponsors ne doivent pas utiliser d'images ou de supports
+      à connotation sexuelle. Ils ne doivent pas non plus se livrer à des activités
+      à connotation sexuelle. L'équipe du stand (y compris les volontaires) ne
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      à connotation sexuelle. L'équipe du stand (y compris les bénévoles) ne
```

File: index-fr.html [hunk 6]
```diff
@@ -14,80 +14,91 @@
...
+      doit pas utiliser de vêtements, uniformes ou costumes à connotation
       sexuelle. Ils ne doivent pas non plus créer un environnement sexualisé.
     </p>
 
     <p>
-      Si un participant a un comportement de harcèlement, les organisateurs de
-      la conférence peuvent prendre toute action qui leur semble adéquate. Cela
-      va d'un simple avertissement à l'exclusion du participant de la conférence
-      sans remboursement.
+      Face à un comportement de harcèlement, les organisateurs de la conférence
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      Face à un comportement de harcèlement, l'équipe d'organisation de la conférence
```

File: index-fr.html [hunk 7]
```diff
@@ -14,80 +14,91 @@
...
+      peuvent prendre toute action qui leur semble adéquate.
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      peut prendre toute action qui lui semble adéquate.
```

File: index-fr.html [hunk 8]
```diff
@@ -14,80 +14,91 @@
...
+      Cela va d'un simple avertissement à l'exclusion sans remboursement de la conférence.
     </p>
 
     <p>
-      Si vous vous sentez harcelé, si vous pensez que quelqu'un se fait
+      Si vous vous sentez harcelé·e, si vous pensez que quelqu'un se fait
       harceler, et plus généralement en cas de problème, merci de contacter
-      immédiatement un membre de l'organisation de l'événement. Les membres sont
-      facilement identifiables à leur t-shirts aux couleurs de l'événement.
+      immédiatement l'organisation de l'événement. L'organisation s'assure
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
      immédiatement l’équipe d’organisation de l'événement. L'organisation s'assure
```

File: index-fr.html [hunk 9]
```diff
@@ -14,80 +14,91 @@
...
+      que ses membres sont facilement identifiables (badge, brassard, tour de cou,
+        t-shirt...) présentés au début de la conférence.
```
Comment (MattiSG) [inline index-fr.html]:
```suggestion
        t-shirt…) présentés au début de la conférence.
```

**Comment by MattiSG** [comment]
The change you outlined in 3b (and possibly 3c) diverges too much from the English version for me.
I don't feel at ease with the French version stating that “organisers **commit to take appropriate sanctions** against those who violate these rules, up to exclusion and legal action” where all other languages write that they **may** do so.

This is a template and each organising team is free to change the language of it, but I believe it is important that translations are just translations, especially for multilingual events.

**Comment by MattiSG** [comment]
My suggestion is to keep the current phrasing with “may” in this PR, and possibly open a separate PR to suggest the change in the English (possibly French at the same time) version so we can discuss this change more broadly 🙂

**Comment by MattiSG** [comment]
@abelards Lovely! Hope these change suggestions were useful 👍 
Is it okay with you if I switch the middle dots to hyphenation points? (reference: https://medium.com/p/3f749c32b659)

**Comment by abelards** [comment]
Thanks for the review/changes @MattiSG !

I am OK with you changing the dots. I'm sad because I googled some resources to copy-paste the middle dot, but I'm glad you have a keener eye ;)

**Comment by MattiSG** [comment]
Awesome, a great improvement for the French translation, congrats @abelards!  😃

**Pull request merged**